File: lat2/cpp/timsepakbola.cpp
/* cpp */

// membuat kelas untuk daftar tim sepakbola 
class DaftarTimSepakbola{

	private: 
		string namaTim;
		string asalNegara;
		int tahunTim;
		string namaPemain;
		int nomorPemain;
		string posisi;

	public:
		// constructor
		DaftarTimSepakbola(){
		}
		
		// set dan get dari private
		void setNamaTim(string namaTim){
			this->namaTim = namaTim;
		}
		string getNamaTim(){
			return this->namaTim;
		}

		void setAsalNegara(string asalNegara){
			this->asalNegara = asalNegara;
		}
		string getAsalNegara(){
			return this->asalNegara;
		}

		void setTahunTim(int tahunTim){
			this->tahunTim = tahunTim;
		}
		int getTahunTim(){
			return this->tahunTim;
		}

		void setNamaPemain(string namaPemain){
			this->namaPemain = namaPemain;
		}
		string getNamaPemain(){
			return this->namaPemain;
		}

		void setNomorPemain(int nomorPemain){
			this->nomorPemain = nomorPemain;
		}
		int getNomorPemain(){
			return this->nomorPemain;
		}

		void setPosisi(string posisi){
			this->posisi = posisi;
		}
		string getPosisi(){
			return this->posisi;
		}



		// destructor
		~DaftarTimSepakbola(){}

};

File: lat2/java/timsepakbola.java
/* Java */

// membuat kelas untuk daftar tim sepakbola 
class DaftarTimSepakbola{

	private String namaTim;
	private	String asalNegara;
	private	int tahunTim;
	private	String namaPemain;
	private	int nomorPemain;
	private	String posisi;

	// constructor
	public DaftarTimSepakbola(){
	}
		
	// set dan get dari private
	public String getNamaTim(){
		return namaTim;
	}
	public void setNamaTim(String namaTim){
		this.namaTim = namaTim;
	}

	public String getAsalNegara(){
		return asalNegara;
	}
	public void setAsalNegara(String asalNegara){
		this.asalNegara = asalNegara;
	}


	public int getTahunTim(){
		return tahunTim;
	}
	public void setTahunTim(int tahunTim){
		this.tahunTim = tahunTim;
	}


	public String getNamaPemain(){
		return namaPemain;
	}
	public void setNamaPemain(String namaPemain){
		this.namaPemain = namaPemain;
	}


	public int getNomorPemain(){
		return nomorPemain;
	}
	public void setNomorPemain(int nomorPemain){
		this.nomorPemain = nomorPemain;
	}


	public String getPosisi(){
		return posisi;
	}
	public void setPosisi(String posisi){
		this.posisi = posisi;
	}
}


